src/plugins/teleport-plugin-react-base-component/index.ts

```diff
@@ -21,6 +21,10 @@ export const createPlugin: ComponentPluginFactory<JSXConfig> = (config) => {
   const reactComponentPlugin: ComponentPlugin = async (structure) => {
     const { uidl, dependencies } = structure
 
+    if (uidl.node.type === 'conditional') {
```

Comment:
Similar things happen when a repeat is the root node. :-? I'm not sure if just a 'conditional' should throw an error. I'm wondering if we could actually have conditionals and repeats supported as roots instead of just elements :-?

Reply:
Yeah, i realised after giving the PR, that it is possible to have ``repeat`` as the root element, but my personal opinion is it would be good if we can support both repeat and conditional as root, by wrapping the component with ``Fragments`` in this specific case, in that way we won't be running into this. Alex said he will be looking into this, let's see 👍